ui/src/app/utils/resourceActions.ts

```diff
@@ -27,6 +27,10 @@ export const ResourceTypeDisplayNames = new Map()
   .set(ResourceType.INVALID, 'Invalid')
   .set(ResourceType.WORKSPACE, 'Workspace');
 
+export function getResourceTypeDisplayName(resourceType: ResourceType) {
```

Comment:
It'd be cool if the enum values themselves supported a method to get the name. Then you could do `myResourceType.getDisplayName()`. 

This is great too though. Don't spend more than a couple of minutes experimenting if you want to try that.

Reply:
so.. after looking at it with fresh eyes, I don't think I see any reason why we need a separate Map or an enum with different values from the display name. So now the enum contains the display names. Any thoughts/objections to this approach?